students/douglas_klos/session3/list_lab.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+# Douglas Klos
+# Python 210, Exercise 3
+# January 24st, 2019
+# list_lab.py
+
+
+def series1():
+    """
+    Creates the specified list and displays it. Asks user for another
+    fruit and adds it to end of list.  Displays list again.  Asks user
+    for a number, prints it back and its coressponding fruit.
+    Adds another fruit to the beginning of list using +.
+    Adds another fruit to the beginning of list using insert.
+    Displays all fruit in list beginning with P.
+    """
+
+    print("Series 1:")
+    fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+    print(fruit)
+
+    # Ask a user for a fruit and add to end of list.
+    fruit += (input("Enter the name of fruit to be added: "),)
+    print(fruit)
+
+    # Ask user for anumber and display the number back to the use and
+    # the fruit corresponding to that number.  No error trapping.
+    index = input("Enter the index of fruit to print: ")
+    print("index = " + index + ", fruit = " + fruit[int(index)-1])
+
+    # Add fruit to beginning using the "+".
+    fruit = [input("Enter fruit to add at beginning of list: ")] + fruit
```

Comment:
Try to use the built in functions of your list object such as `.insert` to insert a value at a given index
`fruit.insert(0, input("Enter fruit to add at beginning of list: "))`
I would probably also break out the `input` command to put into a variable instead of inline with the `.insert` statement just to make it more readable.